portfoliostyle.css

```diff
@@ -0,0 +1,310 @@
+* {
+	Box-sizing: border-box;
+	font-family: monaco;
+	
+}
+
+div {
+	display: flex;
+}
+
+body {
+	/*background-color: #aaabb8;*/
+	background-image: url("images/snow-3161134_1920.jpg");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+}
+
+nav {
+	display: flex;
+	background-color: #25274d;
+	color: #ffffff;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	padding: 0 25px 0 25px;
+	width: 100%;
+}
+
+nav button {
+	margin-left: 10px;
+	padding: 10px;
+	border-radius: 10px;
+	background-color: #29648a;
+	color: #ffffff;
+	border-color: #2e9cca;
+}
+
+main {
+	margin: 0 auto;
+	width: 90%;
+	flex-direction: column;
+	display: flex;
+	
+	font-size: 1.5vw;
+}
+
```

Comment:
Same with whitespace here: think check both html and css for this as will make you code look more condensed and easier to read if you're consistent with spacing

Reply:
Making my spacing more consistent, will try to keep it in mind in the future